extensions-utils/src/dfx.rs

```diff
@@ -7,13 +7,17 @@ use std::ffi::OsStr;
 use std::path::Path;
 use std::process::{self, Command};
 
-/// Calls a bundled command line tool.
+/// Calls a binary from dfx cache.
 ///
 /// # Returns
 /// - On success, returns stdout as a string.
 /// - On error, returns an error message including stdout and stderr.
-#[context("Calling {} CLI, or, it returned an error.", command)]
-pub fn call_bundled<S, I>(dfx_cache_path: &Path, command: &str, args: I) -> anyhow::Result<String>
+#[context("Calling {} CLI failed, or, it returned an error.", command)]
```

Comment:
```suggestion
#[context("Calling {} CLI, or, it returned an error.", command)]
```
It seems like the context indicates that both branches failed